code/modules/mob/living/carbon/human/human.dm

```diff
@@ -535,17 +535,30 @@
 	if (!penetrate_thick)
 		if(above_neck(target_zone))
 			if(head && istype(head, /obj/item/clothing))
-				var/obj/item/clothing/CH = head
-				if (CH.clothing_flags & THICKMATERIAL)
-					. = 0
-		else
-			if(wear_suit && istype(wear_suit, /obj/item/clothing))
-				var/obj/item/clothing/CS = wear_suit
-				if (CS.clothing_flags & THICKMATERIAL)
-					. = 0
-	if(!. && error_msg && user)
-		// Might need re-wording.
-		to_chat(user, "<span class='alert'>There is no exposed flesh or thin material [above_neck(target_zone) ? "on [p_their()] head" : "on [p_their()] body"].</span>")
+				var/obj/item/clothing/head/CH = head
+				if(CH.clothing_flags & THICKMATERIAL)
+					to_chat(user, "<span class='alert'>There is no exposed flesh or thin material on this head!</span>")
```

Comment:
This head should be their head [p_their]

Reply:
Fixed